node/src/actors/json_rpc/api.rs

```diff
@@ -1040,6 +1044,20 @@ pub async fn get_transaction(hash: Result<(Hash,), Error>) -> JsonRpcResult {
         Err(e) => return Err(e),
     };
 
+    // Get current epoch
+    let current_epoch = match EpochManager::from_registry().send(GetEpoch).await {
+        Ok(Ok(current_epoch)) => current_epoch,
+        Ok(Err(e)) => return Err(internal_error(e)),
+        Err(e) => return Err(internal_error(e)),
+    };
```

Comment:
```suggestion
    let current_epoch = match EpochManager::from_registry().send(GetEpoch).await.map_err(|e| internal_error(e))?;
```

Reply:
Clippy didn't suggest any particular improvement on this part, which I just replicated from other parts of the same function. Perhaps it's better to stick to the way that's been proven to work all along.